darts/backtesting/backtesting.py

```diff
@@ -58,6 +82,20 @@ def backtest_forecasting(series: TimeSeries,
         The first prediction time, at which a prediction is computed for a future time
     fcast_horizon_n
         The forecast horizon for the point predictions
+    target_indices
+        In case `series` is multivariate and `model` is a subclass of `MultivariateForecastingModel`,
+        a list of indices of components of `series` to be predicted by `model`.
+    component_index
+        In case `series` is multivariate and `model` is a subclass of `UnivariateForecastingModel`,
+        an integer index of the component of `series` to be predicted by `model`.
+    use_full_output_length
+        In case `model` is a subclass of `TorchForecastingModel`, this argument will be passed along
+        as argument to the predict method of `model`.
+    stride
+        The number of time steps (the unit being the frequency of `series`) between two consecutive predictions.
+    retrain
+        Whether to retrain the model for every prediction or not. Currently only TorchForecastingModel
```

Comment:
You can add backticks around TorchForecastingModel: `

Reply:
done